modules/dnn/include/opencv2/dnn/dnn.hpp

```diff
@@ -1373,7 +1373,11 @@ class CV_EXPORTS_W_SIMPLE TextRecognitionModel : public Model
 
     /**
      * @brief Set the decoding method of translating the network output into string
-     * @param[in] decodeType The decoding method of translating the network output into string: {'CTC-greedy': greedy decoding for the output of CTC-based methods}
+     * @param[in] decodeType The decoding method of translating the network output into string:
+     * {
+     *    'CTC-greedy': greedy decoding for the output of CTC-based methods
+     *    'CTC-prefix-beam-search': Prefix beam search decoding for the output of CTC-based methods
+     * }
```

Comment:
[Documentation](http://pullrequest.opencv.org/buildbot/export/pr/20524/docs/de/dee/classcv_1_1dnn_1_1TextRecognitionModel.html#ad59a4961f3394724d7d7535fff28693d) doesn't look well.

It makes sense to move possible values before the `@param` statement.
Consider using the "list" mode through (`-`) without `{}`

Reply:
Fixed. Format refers to cv::dnn::readNet().